docs/source/dataset_streaming.rst

```diff
@@ -93,6 +96,29 @@ Tokenizers are written in Rust and use parallelism to speed up tokenization. To
     {'input_ids': [101, 11047, 10497, 7869, 2352...], 'token_type_ids': [0, 0, 0, 0, 0...], 'attention_mask': [1, 1, 1, 1, 1...]}
 
 
+Split your dataset with ``take`` and ``skip``
+--------------------------------------------------
+
+You can split your dataset by taking or skipping the first ``n`` examples.
+
+You can create new dataset with the first ``n`` examples using :func:`datasets.IterableDataset.take`, or you can get a dataset with the rest of the examples by skipping the first ``n`` examples with :func:`datasets.IterableDataset.skip`:
+
+
+.. code-block::
+
+    >>> dataset_head = dataset.take(2)
```

Comment:
would it make sense to include the `dataset = load_dataset(...)` line in the example?